VERSION

```diff
@@ -0,0 +1 @@
+v0.1.0
```

Comment:
I think we should get rid of this VERSION file. The os-rootfs repo works fine without it.